content-security-policy/support/testharness-helper.sub.js

```diff
@@ -41,6 +41,13 @@ function generateUrlWithPolicies(host, policy) {
     return url + "?policy=" + encodeURIComponent(policy);
 }
 
+function generateUrlWithAllowCSPFrom(host, allowCspFrom) {
+  var url = generateURL(host, PolicyHeader.ALLOW_CSP_FROM);
+  if (!allowCspFrom)
+    return url + "?allow_csp_from=";
```

Comment:
Also, it's going to look the same to Python. Perhaps not sending `allow_csp_from` at all would be a better way to signal to the backend that it shouldn't set the header at all (as opposed to setting it to an empty value).